spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-reactive-discovery-client-it/src/test/java/org/springframework/cloud/kubernetes/client/reactive/discovery/it/ReactiveDiscoveryClientIT.java

```diff
@@ -31,9 +31,9 @@
 import io.kubernetes.client.openapi.models.V1Service;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.junit.After;
```

Comment:
the changes are transparent here, imho: junit-5, move methods close to each by logic (providers of manifests, `@BeforeClass` and `@AfterClass`), cast to `Map<String, Object>` instead of a raw `Map`, etc. The test has not seen any functional changes at all